### ISSUE TITLE
[Refactor] Make Connect WalletConnect & Ledger keys as flow

### ISSUE THREAD

**Issue by MouazAlzahabi** [safe-global/safe-ios#2770]
We want to make add wallet connect and ledger keys using the flow structure, same as we have in other key types

**Comment by DmitryBespalov** [comment]
@MouazAlzahabi I think this is done already?

**Comment by MouazAlzahabi** [comment]
@DmitryBespalov Yes it's done already, I guess it has been returned by QA because of reported issues like #2773
